### PR TITLE
Upgrade ESMAbstract + changing licensing

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by kmbarry1 — APPROVED** [review]
LGTM

File: src/dapp/DSRuneAbstract.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: AGPL-3.0-or-later
```
Comment (brianmcmichael) [inline src/dapp/DSRuneAbstract.sol:16]: Preexisting issue here, but SPDX header should probably be moved above the copyright block.

**Review by brianmcmichael — APPROVED** [review]
One minor thing, fix it if convenient, but shouldn't block merge.

File: src/dapp/DSRuneAbstract.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/dapp/DSRuneAbstract.sol]: Fixed.

**Pull request merged**